guide/C/appendixa.xml

```diff
@@ -1,73 +1,107 @@
-for a different time, but if you can get one written, here&rsquo;s what you
-need to do:</para>
-  <orderedlist>
-  <listitem>
-    <para>Copy the &app; <acronym>XML</acronym> data file to a working file.</para>
-    <note>
-      <para>If the file was last modified by a version of &app; older than 2.0, then
-before to continue to the next step you will need to modify the working file&rsquo;s &lt;gnc-v2&gt;
-tag to read something like this:</para>
-    <literallayout>
+  <sect1 id="appendixa_accts_vs_cats1">
+    <title>Using Accounts vs. Categories</title>
+    <para> If you are familiar with other personal finance programs, you are already accustomed to
+      tracking your income and expenses as categories. Since &app; is a double-entry system (refer
+      to section 2.1), income and expenses are tracked in accounts. The basic concept is the same,
+      but the account structure allows more consistency with accepted business practices. So, if you
+      are a business user as well as a home user, &app; makes it easy to keep track of your business
+      as well as your personal accounts. 
+    </para>
+    <para>
+      Income and expense accounts give you the same information you would get with categories, but
+      they also give you more flexibility in entering your transactions. In &app;, you have the
+      option to enter transactions directly into income and expense accounts through their account
+      registers. Other programs that use categories do not offer this option, because there is no
+      <quote>account register</quote>
+      for a category.
+    </para>
+    <para>
+      You also have the option in &app; to treat income and expense accounts exactly as you would
+      treat categories, if you are more comfortable with that method. In
+      <trademark class="registered">Quicken</trademark>
+      and similar programs, transactions require an account and a category. Substitute an income or
+      expense account name in &app; where you would normally enter a category name in the other
+      programs, and the result should be the same. We will discuss transaction entry in Chapter 4 in
+      greater detail. 
+    </para>
+  </sect1>
+  <sect1 id="appendixa_qif1">
+    <title>
+      Organization of
+      <acronym>QIF</acronym>
+      Files (Discussion)
+    </title>
+    <para />
+  </sect1>
+  <sect1 id="appendixa_entercharge1">
+    <title>Common Duplication Issues (Discussion)</title>
+    <para />
+  </sect1>
+  <sect1 id="appendixa_enterpay1">
+    <title>
+      Checking
+      <acronym>QIF</acronym>
+      Data (Discussion)
+    </title>
+    <para />
+  </sect1>
+  <sect1 id="appendixa_xmlconvert1">
+    <title>
+      Converting
+      <acronym>XML</acronym>
+      &app; File
+    </title>
+    <para>The
+      &app;
+      <acronym>XML</acronym>
+      data file can be transformed to almost any other data format (e.g.,
+      <acronym>QIF</acronym>
```

Comment:
If you can figure out how it would be good to distinguish style tags like `<acronym>` from formatting tags like `<para>` and block only on formatting tags.

Reply:
Yes, distinguish block elements from inline elements.
I will try to find it in the config or test several tools.